### PR TITLE
[Review] Request from '' @ 'yast/ycp-killer/improve-option-descriptions'

### PR DESCRIPTION
**Pull request by dmajda**
Please review the following changes:
- cdffe49 Improve option descriptions
- ec9ca53 Format option definitions bit more nicely

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,9 +254,9 @@ The `help` command can be used to display a short overview of available commands
       yk ybc <module>...          # Compile module's YCP modules into YCP bytecode
 
     Options:
-      [--debug]      # verbosely log what commands are run
-      [--with-deps]  # also include module dependencies in operations
-      [--threads=N]  # limit the number of threads in parallel tasks (default: all detected CPUs)
+      [--debug]      # Log executed external commands
+      [--with-deps]  # Operate also on dependencies (ybc_deps) of specified modules
+      [--threads=N]  # Use <= N threads for parallelized commands (default: N = number of CPUs)
```
Comment (lslezak) [inline README.md:259]: `Use max. N threads` would be better IMHO.

File: yk [hunk 2]
```diff
@@ -41,9 +41,17 @@ require_relative "lib/commands/test_command"
...
+               :type => :boolean,
+               :desc => "Log executed external commands"
+
+  class_option :with_deps,
+               :type => :boolean,
+               :desc => "Operate also on dependencies (ybc_deps) of specified modules"
+
+  class_option :threads,
+               :type => :numeric,
+               :desc => "Use <= N threads for parallelized commands (default: N = number of CPUs)"
```
Comment (lslezak) [inline yk:54]: The same here...

File: README.md (hunk 1 above)
Comment (dmajda) [inline README.md:259]: I also thought about it, I guess you are right. Fixed in #609.

File: yk (hunk 2 above)
Comment (dmajda) [inline yk:54]: Also fixed in #609.

**Comment by dmajda** [comment]
Obsoleted by #609.

**Pull request closed without merging**